src/components/forms/Switch/Switch.stories.tsx

```diff
@@ -42,3 +39,33 @@ export const Disabled = Template.bind({});
 Disabled.args = {
   isDisabled: true,
 };
+
+export const DisabledChecked = Template.bind({});
+DisabledChecked.args = {
+  isDisabled: true,
+  isSelected: true,
+};
+
+export const Loading = Template.bind({});
```

Comment:
It's really great that you care, but Switch doesn't have the `loading` state.

Reply:
But in real life, it does: 
<img width="229" alt="CleanShot 2022-10-03 at 13 23 46@2x" src="https://user-images.githubusercontent.com/61194245/193543896-530bda7a-b09f-40c7-a8de-d356ed3d261f.png">

Reply:
Then it shouldn't